json-schema/schema.json

```diff
@@ -136,7 +138,7 @@
           }
         },
         "nodata": {
-          "type": "number"
+          "type": ["number","string"]
```

Comment:
Please add the string type in the table of the readme as well

Reply:
Done